robot/Cumulus/resources/locators_49.py

```diff
@@ -260,6 +260,7 @@
         'combobox':'//div[contains(@class,"slds-is-open") and @role="combobox"]',
         'footer-btn':"//footer[@class='slds-modal__footer']/button[text()='{}']",
         'field-label':'//lightning-primitive-cell-factory//*[text()="{}"]',
+        'field_dd_option':'//div[contains(@class,"slds-input-has-icon")][./input[@name="{}"]]/following-sibling::div//span[@title="{}"]'
```

Comment:
This is a personal preference, but I'm not a huge fan of abbreviations unless absolutely necessary. Just so I can skip the tiny bit extra mental translation when I read them. 'dd' to 'dropdown' here as an example here.

Reply:
sure, sounds good. Will make sure to use full names going forward